go.mod

```diff
@@ -2,20 +2,20 @@ module github.com/edgexfoundry/app-service-configurable
 
 go 1.20
 
-require github.com/edgexfoundry/app-functions-sdk-go/v3 v3.0.0-dev.41
+require github.com/edgexfoundry/app-functions-sdk-go/v3 v3.0.0-dev.45
```

Comment:
This version allows the TOMLL module to be removed from the Attribution.txt file